lib/dartino/device/device.dart

```diff
@@ -51,5 +53,71 @@ abstract class Device {
 
   /// Launch the specified application on the device and return `true`.
   /// If there is a problem, notify the user and return `false`.
-  Future<bool> launchSOD(SodRepo sdk, DartinoLaunch launch);
+  Future<bool> launchSOD(SodRepo sdk, DartinoLaunch launch) async {
+    //TODO(danrubel) add windows and mac support and move this into cmdline util
+    if (isWindows && isMac) {
+      atom.notifications.addError('Platform not supported');
+      return false;
+    }
+    // Compile
+    String binPath = await sdk.compile(launch);
+    if (binPath == null) return false;
+    String destPath;
+    destPath = '/spifs/${fs.basename(binPath)}';
+    //TODO(danrubel) remove next line once files can be removed or overwritten
+    // destPath = '/spifs/d${new DateTime.now().millisecondsSinceEpoch}.snap';
+    // Start background daemon if not already running
+    LkShell shell = await sdk.startDebugDaemon(launch);
+    if (shell == null) return false;
+    // Deploy to device
+    shell.write('\n');
+    await new Future.delayed(new Duration(milliseconds: 250));
```

Comment:
This delay is the most expedient way for now. This code will be ripped out once the underlying debug daemon is fixed to handle resetting the device, removing the file, etc.